test/e2e/network/netpol/kubemanager.go

```diff
@@ -115,11 +124,14 @@ func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, container
 	var cmd []string
 	switch protocol {
 	case v1.ProtocolSCTP:
-		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=1s", "--protocol=sctp"}
+		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=3s", "--protocol=sctp"}
```

Comment:
I'd parameterize the timeout so you only have to change it in one place in the future.

Reply:
yes, great idea, just using a constant will be enough

Reply:
fixedddd